Blackjack.Tests/PlayerShould.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using Moq;
+using Xunit;
+
+namespace Blackjack.Tests
+{
+    public class PlayerShould
+    {
+        [Fact]
+        public void Verify_PlayerReceives2Cards()
+        {
+            var mockPlayer = new Mock<IPlayer>();

```

Comment:
Never use mocks for the concrete class that you are testing. What this means is that if you are testing the Player class, then you should be using real player instances and not mocks.
The reason why mocks only work for abstractions (interfaces or abstract classes) is that it just needs to know which methods are public, but it has no knowledge of the code that is inside of those methods. This is why when you use mocks, depending on the usage, you need to specify what value a particular function you are calling on the mock is returning.
In this case where you are testing the player class, you are making sure that the code in each method you are testing is doing what you expect it to - which is why you can't use mock players in a Player test class.